CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+---
+CIP: 88
+Title: Hybrid Token Policy Registration
+Category: Tokens
+Status: Proposed
+Authors:
+    - Adam Dean <adam@crypto2099.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/467
+Created: 2023-02-27
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Currently, token projects (NFT or FT) have no mechanism to register the intent, details, or feature set of their minting
+policy on-chain. This CIP will aim to create a method that is backwards compatible and enable projects to declare, and
+update over time, the supported feature set and metadata details pertaining to their tokens.
+
+This CIP will aim to make use of a hybrid (on and off-chain) information schema to enable maximum flexibility and 
+adaptability as new and novel use cases for native assets (NA) expand and grow over time.
+
+## Motivation
+
+This CIP was borne out of a distaste for the lack of on-chain token policy intent registration that has been cited as
+both a centralization and security concern at various points over the preceding two years of NA history on Cardano.
+
+### Example 1: The Cardano Token Registry
+
+Many Fungible Token (FT) projects require special treatment of their NAs such as decimal places for proper display and
+formatting, project information, and token logo. As it stands, these projects must currently register via a GitHub
+repository ([Cardano Token Registry](https://github.com/cardano-foundation/cardano-token-registry)) in order to have
+their token properly appear in wallets. This GitHub repository is currently managed by the Cardano Foundation (CF).
+While there is no reason to believe that the CF would take any malicious or nefarious action, forcing token projects to
+register in this fashion introduces a point of centralization, _potential_ gate keeping, and ultimately reliance on the
+interaction of a 3rd party (the human at CF responsible for merging pull requests) in order for projects to register, or
+update, their information.
+
+### Example 2: Token Metadata Insecurity
+
+One of the stated rationales for [CIP-68](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0068) was that the
+"original" Cardano NFT Metadata Standard ([CIP-25](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025)) was
+insecure in some example use cases. This is due to the link between the transactional metadata and the minting of native
+assets. For example, a smart contract that cannot/does not validate against transaction metadata (i.e. Liquidity Pool
+tokens) could have a malicious user inject CIP-25 metadata into the transaction, potentially inserting illicit, illegal,
+or otherwise nefarious metadata information tied to the tokens which may be picked up and displayed to end users 
+unwittingly by explorers and wallets.
```

Comment:
Yes, but not _only_. The real security concern regard metadata spoofing. CIP-0025 supposes that EVERY token minting will follow CIP-0025 as a metadata process. If a project decides not to, then there's nothing preventing some user to provide its own metadata as-if the project was CIP-0025 compliant. 

Which means that fundamentally, there's a need for a meta-metadata which informs what standard does a project actually follow. I am sensing that this is possibly CIP-88?

Reply:
That is correct, CIP-88 aims to address the need to explicitly signal support for CIP-25 so that explorers ***may*** refuse to display any CIP-25 metadata found on-chain if a CIP-88 registration is not found.

Reply:
@KtorZ @Crypto2099 I'm resolving this because the purpose of CIP-88 seems explicitly covered in this section; please feel free to re-open if people still think it needs clarification.